features/developer_portal/admin/account/personal_details.feature

```diff
@@ -0,0 +1,36 @@
+Feature: Dev Portal Buyer Personal Details
+  As a buyer
+  I want to change my personal details
+
+  Background:
+    Given a buyer logged in to a provider
+
+  Scenario: Buyer wants to update information without current password
+    Given the buyer wants to edit their personal details
+    When the buyer writes a new name
+    And the buyer writes a new email
+    And clicks on update personal details
+    Then they should see current password is incorrect
+
+  Scenario: Buyer wants to update information adding their current password
+    Given the buyer wants to edit their personal details
+    When the buyer writes a new name
+    And the buyer writes a new email
+    And the buyer writes their current password
+    And clicks on update personal details
+    Then they should see their information updated
+
+  Scenario: Buyer wants to update their password without current password
+    Given the buyer wants to edit their personal details
+    When the buyer writes a new password
+    And the buyer confirms their new password
+    And clicks on update personal details
+    Then they should see current password is incorrect
+
+  Scenario: Buyer wants to update their password adding their current password
+    Given the buyer wants to edit their personal details
+    When the buyer writes their current password
+    And the buyer writes a new password
+    And the buyer confirms their new password
+    And clicks on update personal details
+    Then they should have their password updated
```

Comment:
I'm afraid this is all testing the same, IMO there are 2 possible scenarios: with and without current password. You could go the brief way, something like:
```suggestion
  Scenario: Buyer uses the wrong current password
    Given the buyer wants to edit their personal details
    When they change their personal details and password
    But they add the wrong current password
    Then they should not be able to edit their personal details
    
  Scenario: Buyer uses the wrong current password
    Given the buyer wants to edit their personal details
    When they change their personal details and password
    And they add the correct current password
    Then they should be able to edit their personal details
```

Or the long way, reusing the steps available, such as:
```cucumber
And go to the personal details page
And fill in "Username" with "Alfred"
And press on "Update Personal Details" 
```
(See features/step_definitions/web_steps.rb)

Reply:
What should be tested here though is other validations, such as, sending empty form, sending an invalid email, wrong confirmation password.